acceptance/managers/buildpack_manager.go

```diff
@@ -0,0 +1,42 @@
+// +build acceptance
+
+package managers
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/buildpacks/pack/acceptance/assertions"
+	"github.com/buildpacks/pack/acceptance/components"
+
+	"github.com/buildpacks/pack/internal/api"
+	"github.com/buildpacks/pack/internal/builder"
+)
+
+type BuildpackManager struct {
+	testObject *testing.T
+	assert     assertions.AssertionManager
+	source     string
+}
+
+func NewBuildpackManager(t *testing.T, assert assertions.AssertionManager) BuildpackManager {
+	buildpackAPI := api.MustParse(builder.DefaultBuildpackAPIVersion).String()
+
+	return BuildpackManager{
+		testObject: t,
+		assert:     assert,
+		source:     filepath.Join("testdata", "mock_buildpacks", buildpackAPI),
+	}
+}
+
+func (b BuildpackManager) PlaceBuildpacksInDir(destination string, buildpacks ...components.TestBuildpack) {
```

Comment:
I felt like this name was a bit odd; I would suggest something more like `CopyBuildpacksToDir`

Reply:
The trouble with this is there are other ways of preparing buildpacks than copying. Adding this interface is intended to simplify testing all the different ways buildpacks can be prepared. I agree it's a bit weird though, the directory that's passed in is ignored for package buildpacks, and may not make sense for other buildpacks. Because the directories are generally made after the buildpacks are instantiated it makes it kind of hard to get it to the buildpack concrete implementation at the time it's needed while keeping the objects immutable. Happy to explore alternative patterns, or even an alternative name, but I don't think it should be `Copy`